api-definitions/db_positions/api.yaml

```diff
@@ -234,12 +355,14 @@ definitions:
     type: string
     pattern: '[a-zA-Z]{2}[0-9]{2}[a-zA-Z0-9]{1,30}'
     example: IT60X0542811101000000123456
+    maxLength: 255
```

Comment:
is better to work on regex ex `^[a-zA-Z]{2}[0-9]{2}[a-zA-Z0-9]{1,30}$`

Reply:
Deleted maxLength option